### DISCUSSION TITLE
Recommendation for lists within grids

### DISCUSSION THREAD

**Discussion by ofek** [squidfunk/mkdocs-material#6909] [Q&A]
### Bug description

I'm trying to use a list within the grid element and the rendering is mistaken for nesting:

![image](https://github.com/squidfunk/mkdocs-material/assets/9677399/66127e84-f56f-48ba-8596-4214875b7042)

### Related links

https://squidfunk.github.io/mkdocs-material/reference/grids/#list-syntax

### Reproduction

N/A, for now

### Steps to reproduce

````markdown
<div class="grid cards" markdown>

-   Markdown

    ---

    ```markdown
    1. first line

         second line

    1. third line
    ```

-   Rendered

    ---

    1. first line

         second line

    1. third line

</div>
````

### Browser

Chrome

### Before submitting

- [X] I have read and followed the [bug reporting guidelines](https://squidfunk.github.io/mkdocs-material/contributing/reporting-a-bug/).
- [X] I have attached links to [the documentation](https://squidfunk.github.io/mkdocs-material/), and possibly related [issues](https://github.com/squidfunk/mkdocs-material/issues) and [discussions](https://github.com/squidfunk/mkdocs-material/discussions).
- [X] I assure that I have [removed all customizations](https://squidfunk.github.io/mkdocs-material/contributing/reporting-a-bug/#remove-customizations) before submitting this bug report.
- [ ] I have attached a __.zip file__ with a [minimal reproduction](https://squidfunk.github.io/mkdocs-material/guides/creating-a-reproduction/) using the [built-in info plugin](https://squidfunk.github.io/mkdocs-material/plugins/info/).

**Comment by squidfunk** [comment]
Thanks for asking. This is essentially a question and not a bug report, as you did not provide a minimal reproduction, so it is unclear whether it's actually a bug, or undesired behavior resulting from specific configuration. Converting to discussion.

**Comment by kamilkrzyskow** [comment]
Hello @ofek,
this seems intentional, or a bug as you said 🤔This CSS is applied:
```css
.md-typeset ol ol, .md-typeset ul ol {
  list-style-type: lower-alpha;
}
```
Specifically the second selector: `.md-typeset ul ol`, which is quite general, so perhaps there should be an exception for grids. 
I also feel that an ordered list inside an unordered list should not have the `list-style-type: lower-alpha;` style, but it should start off with decimals hmm. But it's up to @squidfunk to decide.

You can override it with https://squidfunk.github.io/mkdocs-material/customization/#additional-css for the time being to `list-style-type: decimal;`

**Reply by kamilkrzyskow** [reply]
As a side note, I didn't manage to render the code block as in the example above:
![image](https://github.com/squidfunk/mkdocs-material/assets/34622465/698a0c3d-8532-4880-b4fb-cf44cea1cda0)
[9.5.13-example.zip](https://github.com/squidfunk/mkdocs-material/files/14604902/9.5.13-example.zip)

**Reply by squidfunk** [reply]
Thanks for investigating @kamilkrzyskow. It could be a bug yes, but to be sure, we need a self-contained minimal reproduction. The issue only referenced Markdown, which is why its incomplete. @ofek if you can provide something that we can run, we can investigate. Otherwise, we'll be guessing your configuration. Please, always, always provide minimal reproductions when opening issues. It is absolutely essential. Thank you.
